cves/kernel/CVE-2021-45486.yml

```diff
@@ -159,10 +166,14 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Linux kernel was slowing down becaue it cold not effeciently locate 
+    trusted ipv4 addresses, so a hash table was created for easy access. 
+    However, the hash table was not large enough in size, so the size of the table
+    is dynamically allocated based on several factors specific to the machine.
```

Comment:
While this explains why the issue was fixed, it does not fully explain how the vulnerability was found.
From reading the commit message of the fix, it seems that the developer discovered that the hash table they used was too small although I could not find any more information regarding how they determined the table was too small.

Reply:
I also could not find any evidence as to how the vulnerability was found